pkg/reconciler/instances/connectivityproxy/command_test.go

```diff
@@ -24,6 +25,9 @@ import (
 )
 
 func TestCommand(t *testing.T) {
+	err := os.Setenv("GIT_CLONE_TOKEN", "token") //#nosec [-- Ignore nosec false positive. It's not a credential, just an environment variable name]
```

Comment:
Replace it with t.Setenv("GIT_CLONE_TOKEN", "token")

Reply:
Done